webserver.Tests/Project/Controllers/ClientControllerTest.cs

```diff
@@ -0,0 +1,261 @@
+using Microsoft.Data.Sqlite;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using webserver.Data;
+using webserver.Models;
+using webserver.Models.DTOs;
+using webserver.Controllers;
+using Newtonsoft.Json;
+using NuGet.Protocol;
```

Comment:
Why do you need this?

Reply:
NuGet.Protocol was added by VS Code during coding, i removed. But the rest are needed for Models and Json Serialization